doc/nrf/device_guides/working_with_nrf/nrf70/developing/scan_operation.rst

```diff
@@ -0,0 +1,313 @@
+   :depth: 2
+
+This guide demonstrates how the scan operation can be optimized on the nRF70 Series ICs.
+
+Wi-Fi scanning
+**************
+
+Scanning is the process by which a Wi-Fi® device can discover Access Points (APs) in its vicinity.
+The device can use the results of the scan to:
+
+* Choose a suitable AP to connect to.
+
+* Identify a probable candidate AP to roam to in the future.
+
+* Provide location services.
+
+Scanning can be done using one of the following methods:
+
+* **Active:** During an active scan, the device transmits probe requests on different channels and listens for a probe response from APs.
+
+* **Passive:** During a passive scan, the device listens for beacons sent periodically by APs on different channels.
+
+Active scans are quicker than passive scans.
+However, active scans require more instantaneous power for transmitting probe requests.
+
+Regulatory domain rules enforce what channels can be scanned as well as whether an active scan can be performed on a particular channel.
+For more information on regulatory support, see :ref:`ug_nrf70_developing_regulatory_support`.
+
+Active scan on a channel is preferable if:
+
+* Allowed by regulatory rules
+
+* The device can source the required power to transmit probe requests
+
+If either of the above conditions cannot be met, then passive scans have to be used.
+
+Scan operation with nRF70 Series ICs
+************************************
+
+The nRF70 Series ICs can be used as scan-only devices and are an ideal solution for applications that require accurate location information based on Wi-Fi network scanning.
+They provide a reduced and more deterministic code memory footprint by limiting the functionality to only scanning.
+
+The nRF70 Series ICs control the scan operation using various controls, such as:
+
+* Band filtering: Controls the Wi-Fi frequency bands to be scanned (for example, 2.4 GHz, 5 GHz, or both) on the nRF7000 and nRF7002 ICs.
+
+* Forced passive scan: Forces a passive scan on all channels.
+
+* Active dwell time control: Controls the time spent scanning on an active channel.
+
+* Passive dwell time control: Controls the time spent scanning on a passive channel.
+
+* Channel control: Controls the channels to be scanned.
+
+* SSID filtering: Controls the SSIDs to be scanned.
+
+* Scan result limit control: Controls the number of scan results returned to an application.
+
+The above controls can be used individually or in combination to achieve:
+
+* Optimized scanning behavior - It can be used by applications based on their requirements and the conditions of the environment where an end product is expected to be deployed.
+
+* Reduced processing overhead
+
+* Faster scan times
+
+* Reduced power consumption
+
+Scan API
+********
+
+The scan API can be used to implement features such as scan functionality and controlled scan provided by the nRF70 Series ICs.
+
+You can find the scan API reference in :ref:`zephyr:wifi_mgmt` in the Zephyr documentation.
```

Comment:
This should be pointing to https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-12233/zephyr/connectivity/networking/api/wifi.html#c.wifi_mgmt_ops.scan